pkg/cmd/controller/image.go

```diff
@@ -128,12 +127,12 @@ func (u podSpecUpdater) Update(obj runtime.Object) error {
 	case *kbatchv1.Job:
 		_, err := u.kclient.BatchV1().Jobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
 		return err
+	case *kbatchv1.CronJob:
+		_, err := u.kclient.BatchV1().CronJobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
+		return err
 	case *kbatchv1beta1.CronJob:
 		_, err := u.kclient.BatchV1beta1().CronJobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
 		return err
-	case *kbatchv2alpha1.CronJob:
-		_, err := u.kclient.BatchV2alpha1().CronJobs(t.Namespace).Update(context.TODO(), t, kmetav1.UpdateOptions{})
-		return err
```

Comment:
@dmage @ricardomaraschini @openshift/openshift-team-build-api 

CronJob v2alpha1 is removed in v1.21